### DISCUSSION TITLE
Better way of handling configuration

### DISCUSSION THREAD

**Discussion by jolitp** [NTBBloodbath/doom-nvim#395] [Q&A] [answered]
How is the best way to handle custom config? I saw in an issue that I could put the config files in a folder separate from the distribution, like how Doom-Emacs work, but could not find that in the documentation.

**Comment by NTBBloodbath** [comment, marked as answer]
Hey,

Seems like we didn't port all of our previous documentation files to the new Doom version (_and I gotta fix that, my apologies!_). However, you can take a look at the old FAQ [here](https://github.com/NTBBloodbath/doom-nvim/blob/develop/docs/faq.md#how-to-version-control-doom-nvim) as it does have a section about this.

There's also [this small code chunk](https://github.com/NTBBloodbath/doom-nvim/blob/891d7566cd9bfa7445d689e598abaa68d922e88f/lua/doom/core/system.lua#L16-L17) that tells you what directory is used by Doom-Nvim to look for your custom configuration files (and you can also tweak it to point to elsewhere you wish and should work as expected).

### **tl;dr**

Doom-Nvim follows the same conventions as Doom-Emacs to handle custom config by default, that means, Doom-Emacs does make use of `~/.config/doom` directory to store user configurations and Doom-Nvim does make use of `~/.config/doom-nvim` directory for that purpose.

You can store `config.lua` and `modules.lua` configuration files into that directory and Doom-Nvim will look for them first by using a precedence order to load its configuration files.
